From 2f36e177224a0681671145b2b1e12eb2dfe9930f Mon Sep 17 00:00:00 2001
From: Aristeu Rozanski <aris@redhat.com>
Date: Thu, 10 Apr 2014 19:29:26 -0400
Subject: [edac] sb_edac: mark MCE messages as KERN_DEBUG

Message-id: <20140410192925.209446763@napanee.usersys.redhat.com>
Patchwork-id: 78861
O-Subject: [RHEL7.1 PATCH v2 2/2] sb_edac: mark MCE messages as KERN_DEBUG
Bugzilla: 1056785
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1056785
brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7325861
testing: by me in a beaker system
upstream: 49856dc973cd95d85ac1cab6c70410d8331e5c04

 commit 49856dc973cd95d85ac1cab6c70410d8331e5c04
 Author: Aristeu Rozanski <arozansk@redhat.com>
 Date:   Tue Mar 11 15:45:42 2014 -0400

     sb_edac: mark MCE messages as KERN_DEBUG

     Since the driver is decoding the MCE, it's useless to have these
     messages printed unless you're debugging a problem in the driver.

     Signed-off-by: Aristeu Rozanski <arozansk@redhat.com>
     Signed-off-by: Mauro Carvalho Chehab <m.chehab@samsung.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/edac/sb_edac.c b/drivers/edac/sb_edac.c
index a32fed9..6903305 100644
--- a/drivers/edac/sb_edac.c
+++ b/drivers/edac/sb_edac.c
@@ -1849,17 +1849,18 @@ static int sbridge_mce_check_error(struct notifier_block *nb, unsigned long val,
  else
   type = "Event";
 
- printk("sbridge: HANDLING MCE MEMORY ERROR\n");
-
- printk("CPU %d: Machine Check %s: %Lx Bank %d: %016Lx\n",
-        mce->extcpu, type, mce->mcgstatus, mce->bank, mce->status);
- printk("TSC %llx ", mce->tsc);
- printk("ADDR %llx ", mce->addr);
- printk("MISC %llx ", mce->misc);
-
- printk("PROCESSOR %u:%x TIME %llu SOCKET %u APIC %x\n",
-  mce->cpuvendor, mce->cpuid, mce->time,
-  mce->socketid, mce->apicid);
+ sbridge_mc_printk(mci, KERN_DEBUG, "HANDLING MCE MEMORY ERROR\n");
+
+ sbridge_mc_printk(mci, KERN_DEBUG, "CPU %d: Machine Check %s: %Lx "
+     "Bank %d: %016Lx\n", mce->extcpu, type,
+     mce->mcgstatus, mce->bank, mce->status);
+ sbridge_mc_printk(mci, KERN_DEBUG, "TSC %llx ", mce->tsc);
+ sbridge_mc_printk(mci, KERN_DEBUG, "ADDR %llx ", mce->addr);
+ sbridge_mc_printk(mci, KERN_DEBUG, "MISC %llx ", mce->misc);
+
+ sbridge_mc_printk(mci, KERN_DEBUG, "PROCESSOR %u:%x TIME %llu SOCKET "
+     "%u APIC %x\n", mce->cpuvendor, mce->cpuid,
+     mce->time, mce->socketid, mce->apicid);
 
  /* Only handle if it is the right mc controller */
  if (cpu_data(mce->cpu).phys_proc_id != pvt->sbridge_dev->mc)
-- 
1.7.1